src/scripts/creature-stats.js

```diff
@@ -545,9 +545,8 @@ function getCustomActionsData() {
 
         if (rolls) {
             for (let dmg of rolls) {
```

Comment:
This is cool and still works, but you could also use array methods with a closure if you wanted. This also makes it easier to abstract if you wanted to modify your closure to a named function.
```
roles.forEach((dmg) => {...
```